src/pokedex.c

```diff
@@ -100,14 +179,1123 @@ extern void DisableNationalPokedex(void);
 extern void sub_805469C(void);
 extern u16 HoennToNationalOrder(u16);
 extern u16 NationalToHoennOrder(u16);
-extern u16 gPokedexOrder_Alphabetical[];
-extern u16 gPokedexOrder_Weight[];
-extern u16 gPokedexOrder_Height[];
 
-u16 NationalPokedexNumToSpecies(u16);
-
-//  asm/pokedex_area_screen
-void ShowPokedexAreaScreen(u16 species, u8 *string);
+static const u16 sPokedexSearchPalette[] = INCBIN_U16("graphics/pokedex/search.gbapal");
+static const u16 sNationalPokedexPalette[] = INCBIN_U16("graphics/pokedex/national.gbapal");
+const u8 gEmptySpacce_839F7FC[0xA4] = {0};
+static const u8 gUnknown_0839F8A0[] = INCBIN_U8("graphics/pokedex/pokedex_cry_layout.bin.lz");
+static const u8 gUnknown_0839F988[] = INCBIN_U8("graphics/pokedex/pokedex_size_layout.bin.lz");
+static const u8 gUnknown_0839FA7C[] = INCBIN_U8("graphics/pokedex/noball.4bpp.lz");
+#include "data/pokedex_orders.h"
+static const struct OamData gOamData_83A0404 =
+{
+    .y = 160,
+    .affineMode = 0,
+    .objMode = 0,
+    .mosaic = 0,
+    .bpp = 0,
+    .shape = 0,
+    .x = 0,
+    .matrixNum = 0,
+    .size = 0,
+    .tileNum = 0,
+    .priority = 1,
+    .paletteNum = 0,
+    .affineParam = 0,
+};
```

Comment:
It might be more readable if the 0 fields are removed:

```c
static const struct OamData gOamData_83A0404 =
{
    .y = 160,
    .priority = 1,
};
```